src/cephadm/cephadm

```diff
@@ -716,7 +911,7 @@ class FileLock(object):
         #   https://github.com/benediktschmitt/py-filelock/issues/31
         #   https://stackoverflow.com/questions/17708885/flock-removing-locked-file-without-race-condition
         fd = self._lock_file_fd
-        self._lock_file_fd = None
+        self._lock_file_fd = -1
         fcntl.flock(fd, fcntl.LOCK_UN)
```

Comment:
Do you know, if `flock` behaves differently when called with `-1` instead of `None`?

Reply:
In both cases return an error, because it expects a file descriptor which is a small, non negative integer.